dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -591,8 +592,8 @@ public void testSearch () throws Exception {
 
         //*****************************************************************
         //Verify if we already have and site search index, if not lets create one...
-        IndiciesAPI.IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
-        String currentSiteSearchIndex = indiciesInfo.site_search;
+        IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+        String currentSiteSearchIndex = indiciesInfo.getSiteSearch();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'currentSiteSearchIndex' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)